packages/design-system/scripts/cmsds-migrate/src/lib/migrate-helpers.mjs

```diff
@@ -0,0 +1,173 @@
+import chalk from 'chalk';
+import { globby } from 'globby';
+import inquirer from 'inquirer';
+import path from 'path';
+import { readdir, readFile, writeFile } from 'node:fs/promises';
+
+// confirm prompt for starting operations, display warning and y/n dialog
+export const confirmStart = () => {
+  return new Promise((resolve, reject) => {
+    inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'yesno',
+        prefix: `${chalk.red('!!')} Please be sure you are running this in a git repo. ${chalk.red('!!\n!!')} Modifications will occur to files listed above.    ${chalk.red('!!\n\n')}${chalk.green('?')}`,
```

Comment:
Call me paranoid. I like where you were going with this, and I want to make it more noticable and include an explicit disclaimer.

```suggestion
                prefix: `
${chalk.bgRed('┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━  !!! CAUTION !!!  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓')}
${chalk.bgRed('┃')}  Please be sure you are running this in a code repository where changes can be rolled back.  ${chalk.bgRed('┃')}
${chalk.bgRed('┃')}          Modifications will occur to files listed above. Proceed at your own risk.           ${chalk.bgRed('┃')}
${chalk.bgRed('┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛')}\n
`,
```

Reply:
I also noticed that prettier isn't being run on these files. That should probably get fixed, and we can prettier-ignore this block.